### PR TITLE
Add PartAngleDisplay.netkan

### PR DESCRIPTION
**Pull request by Darael**
To include the Part Angle Display mod by @Gerry1135

### CODE REVIEW

File: NetKAN/PartAngleDisplay.netkan [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "PartAngleDisplay",
+    "$kref"        : "#/ckan/github/Gerry1135/PartAngleDisplay",
+    "name"         : "Part Angle Display",
+    "abstract"     : "This is a simple KSP plugin that allows you to surface attach parts at accurate angles.",
+    "license"      : "MIT",
+    "resources": {
+        "homepage"    : "http://forum.kerbalspaceprogram.com/index.php?/topic/73551-110-part-angle-display-including-seamless-highly-accurate-rotation-v0311",
+        "respository" : "https://github.com/Gerry1135/PartAngleDisplay"
```
Comment (Gerry1135) [inline NetKAN/PartAngleDisplay.netkan]: Should this be "repository" or is a typo in CKAN?

Comment (plague006) [inline NetKAN/PartAngleDisplay.netkan]: It should be `repository` without the typo.

**Comment by Darael** [comment]
Good catch.  Corrected.

**Comment by plague006** [comment]
Looks, merging with my thanks to both of you.

**Pull request merged**